docker/worker/worker.py

```diff
@@ -572,6 +572,20 @@ def handle_timeout(self, subscriber, subscription, ack_id, message):
     if task_type in ('fixed', 'regressed'):
       oss_fuzz.handle_timeout(task_type, source_id, self._oss_fuzz_dir, message)
 
+  def _log_task_latency(self, message):
+    """Determine how long ago the task was requested.
+
+    Log how long it took to be serviced."""
+    request_time = message.attributes.get('timestamp')
+    if request_time:
+      request_time = int(request_time)
+      latency = request_time = int(time.time())
```

Comment:
should this be int(time.time()) - request_time ?

Reply:
Sigh. Yes. Yes, it should be. Fixed in 2af2b3c